server/command_test.go

```diff
@@ -335,6 +335,57 @@ Triggers:
 
 func TestAddWebhookCommand(t *testing.T) {
 	for _, test := range addWebhookCommandTests {
+		t.Run(test.testName, func(t *testing.T) {
+			p := new(Plugin)
+
+			mockCtrl := gomock.NewController(t)
+			defer mockCtrl.Finish()
+			mockedClient := mocks.NewMockGitlab(mockCtrl)
+
+			if test.scope == "group" {
+				mockedClient.EXPECT().NewGroupHook(gomock.Any(), gomock.Any(), gomock.Any()).Return(test.webhook, nil)
+			} else {
+				project := &gitLabAPI.Project{ID: 4}
+				mockedClient.EXPECT().GetProject(gomock.Any(), gomock.Any(), gomock.Any()).Return(project, nil)
+				mockedClient.EXPECT().NewProjectHook(gomock.Any(), gomock.Any(), gomock.Any()).Return(test.webhook, nil)
+			}
+			p.GitlabClient = mockedClient
+
+			api := &plugintest.API{}
+			conf := &model.Config{}
+			conf.ServiceSettings.SiteURL = &test.siteURL
+			api.On("build", mock.Anything, mock.Anything).Return(conf)
```

Comment:
You need to mock the `getConfig` call here: 
```suggestion
			api.On("getConfig").Return(conf)
```